gradle.properties

```diff
@@ -3,7 +3,7 @@ org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
 	# check these on https://fabricmc.net/develop
-	minecraft_version=1.20
+	minecraft_version=1.20.1
 	yarn_mappings=1.20+build.1
```

Comment:
If you bump the game version, you should also update Yarn to the latest version for that MC version.